### PR TITLE
ci: configure bundlesize

### PR DESCRIPTION
**Pull request by Haroenv**
on purpose expecting this branch to fail before #635

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +46,7 @@
   },
   "dependencies": {
     "algoliasearch-helper": "^2.26.1",
+    "bundlesize": "^0.17.1",
```
Comment (francoischalifour) [inline package.json]: This should be in `devDependencies`.

File: package.json [hunk 2]
```diff
@@ -52,6 +53,7 @@
     "vue": "^2.4.0"
   },
   "devDependencies": {
+    "bundlesize": "0.17.1",
```
Comment (francoischalifour) [inline package.json]: It should be right after `babel-preset-es2015`.

Comment (Haroenv) [reply package.json]: I expected yarn to order this and didn't check :(

File: package.json [hunk 3]
```diff
@@ -32,7 +32,8 @@
     "docs:build:dev": "cd docs && yarn && yarn build:dev",
     "docs:watch": "cd docs && yarn && yarn dev",
     "docs:deploy": "cd docs && yarn && yarn deploy",
-    "test": "jest --coverage && yarn run lint",
+    "test": "jest --coverage && yarn run lint && yarn run test:size",
```
Comment (francoischalifour) [inline package.json]: Probably aside from this PR but I think that the `test` script owns too many responsibilities.

Comment (Haroenv) [reply package.json]: Fairly nitpicky, but this is on purpose. Running `yarn test` will verify everything, if you just want to run the tests, I always just run `yarn jest`. We can align on that over all libraries though

**Pull request merged**